bin.js

```diff
@@ -74,8 +74,10 @@ function getFiles (done) {
 getFiles(function (err, files) {
   if (err) return error(err)
   files.forEach(function (file) {
-    file.data = fmt.transform(file.data)
-    processFile(file)
+    try {
+      file.data = fmt.transform(file.data)
+      processFile(file)
+    } catch (e) {error(file.name + ': ' + e)}
```

Comment:
I ran this file through `standard-format -w` and the `git diff`returned this:

```
@@ -77,7 +77,7 @@ getFiles(function (err, files) {
     try {
       file.data = fmt.transform(file.data)
       processFile(file)
-    } catch (e) {error(file.name + ': ' + e)}
+    } catch (e) {error(file.name + ': ' + e) }
   })
```

Why would it add a space after the last `)` but not move the entire `error(file.name + ': ' + e)` block onto a separate line?